app.py

```diff
@@ -54,6 +67,21 @@ def load_post_ymls_by_tag(tag) -> dict:
 
 @dataclass
 class Post:
+
+    """Class representing a single post
+
+    Attributes:
+        content (str): HTML containing the body of the post
+        date (datetime.datetime): date the post is published
+        excerpt (str): content capped to 3000 chars
```

Comment:
The number 3000 is a detail, probably don't want to specify it here (also because then you now have to remember to change it in two places, if it changes). Rather, it makes more sense to describe the *function* of the attribute, which is more like "a short preview of the post (used in the index page in place of the full post)"